ambry-api/src/main/java/com/github/ambry/config/MySqlAccountServiceConfig.java

```diff
@@ -29,27 +27,39 @@
   public static final String UPDATE_DISABLED = MYSQL_ACCOUNT_SERVICE_PREFIX + "update.disabled";
   private static final String MAX_BACKUP_FILE_COUNT = MYSQL_ACCOUNT_SERVICE_PREFIX + "max.backup.file.count";
 
-  // TODO: Might need to take an array of URLs which would have one write (master) and multiple read urls (backup)
   /**
-   * URL of the mysql database.
+   * Serialized json containing the information about all mysql end points. This information should be of the following form:
+   * <pre>
+   * {
+   *   "dbInfo" : [
```

Comment:
Minor: wondering if we need the top level field or just use a simple array.  If zkInfo does the same thing we can keep it consistent.